client/src/main/java/se/fortnox/reactivewizard/client/HttpClientConfig.java

```diff
@@ -52,6 +52,9 @@
     @JsonProperty("validateCertificates")
     private boolean isValidateCertificates = true;
 
+    private long connectionMaxIdleTimeInMs         = TimeUnit.MILLISECONDS.convert(10, MINUTES);
```

Comment:
so what does this actually affect? That a connection may be inactive for 10 minutes before the pod marks itself as unhealthy?

Reply:
Does that mean that a low traffic service that doesn't get more than one call every 10 minutes will get killed due to inactivity?

Reply:
This is just a way to clean up idle connections, and has nothing to do with unhealthyness.
So if a connection is created, it will be usable for 10 minutes. If noone uses it for 10 minutes, it will will be removed from the pool and when the next request comes it will create a new connection to be put in the pool.

It has nothing with the unhealthyness to do. 
I introduced again, we had it before, it because the tests would not behave the way I wanted otherwise.
And before we had a cleanup interval for 10 minutes.